client/src/components/food/FoodCategories/index.js

```diff
@@ -63,25 +68,34 @@ function FoodCatagories() {
           src="/foodCategory/logo512.png"
         />
       </Hidden>
-      <Container key="1" direction="column" itemColumns="12" spacing={4}>
-        {[
-          { label: 'Breakfast', id: 1 },
-          { label: 'Lunch', id: 2 },
-          { label: 'Dinner', id: 3 },
-          { label: 'Snacks', id: 4 },
-        ]
-          .filter(({ id }) =>
-            matchSmallScreen ? showIcon.id === 0 || showIcon.id === id : id
-          )
-          .map(({ id, label }) => (
-            <FoodCategoryButton
-              key={id}
-              label={label}
-              showIcon={showIcon.foodCategory !== label}
-              handleClick={handleShowSelectedFood(label, id)}
-            />
-          ))}
-      </Container>
+
+      <DailyUserCalories
+        key="5"
+        showCard
+        showChart
+        displayNone={!matchSmallScreen}
+      />
+      <div>
+        <Container key="1" direction="column" itemColumns="12" spacing={4}>
+          {[
+            { label: 'Breakfast', id: 1 },
+            { label: 'Lunch', id: 2 },
+            { label: 'Dinner', id: 3 },
+            { label: 'Snacks', id: 4 },
+          ]
```

Comment:
It might be helpful to pull this hard-coded array:
```
[
  { label: 'Breakfast', id: 1 },
  { label: 'Lunch', id: 2 },
  { label: 'Dinner', id: 3 },
  { label: 'Snacks', id: 4 },
]
```
out into a new file and export it.  For example, create a `mealConstants.js` and export it:
```
export const mealsArray = [
  { label: 'Breakfast', id: 1 },
  { label: 'Lunch', id: 2 },
  { label: 'Dinner', id: 3 },
  { label: 'Snacks', id: 4 },
]
```
.. because it will probably be useful to reuse in other files (and then this file's render code becomes a little shorter, making it easier to read).